src/resources/SchemaService/SchemaService.ts

```diff
@@ -45,4 +46,37 @@ export default class SchemaService extends Ressource {
     create(source: New<CreateSchemaSourceModel, 'resourceId'>, options?: CreateSchemaSourceOptions) {
         return this.api.post<{id: string}>(this.buildPath(SchemaService.baseUrl, options), source);
     }
+
+    translateToSpecificObjectsToGet(
+        sourceType: SourceType,
+        genericObjectsToGet: ObjectsToGet,
+        parameters?: SchemaServiceQueryParams
+    ) {
+        return this.api.post<any>(
+            this.buildPath(`${SchemaService.baseUrl}/${sourceType}/translate/specific`, parameters),
+            genericObjectsToGet
+        );
+    }
+
+    translateToSpecificObjectsToGetWithFields(
+        sourceType: SourceType,
+        genericObjectsToGet: ObjectsToGet,
+        parameters?: SchemaServiceQueryParams
+    ) {
+        return this.api.post<any>(
```

Comment:
Can't we narrow down a bit what we return? `any` is really generic

Reply:
Yes, because every sourcetype can have their own format of Specific Objects to Get, we can't know the shape of the object...
However these routes are defined here `in case` we need them in the future, we don't have a plan to use them straight from the admin-ui for now